dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/preheat/mappers/RelationshipMapper.java

```diff
@@ -64,5 +64,6 @@ public interface RelationshipMapper extends PreheatMapper<Relationship> {
   @Mapping(target = "name")
   @Mapping(target = "attributeValues")
   @Mapping(target = "bidirectional")
+  @Mapping(target = "sharing")
```

Comment:
Why are we not doing this elsewhere @enricocolasante ? And why do the other security validators work 🤷🏻 We discussed this a while back but I don't remember what the outcome was.

Reply:
I am not sure I understand what you mean by elsewhere.
We are mapping `sharing` for other metadata, like `TrackedEntityType` or `Program`

Reply:
I think @teleivo meant why are we not using the `RelationshipMapperType` like we do in the `TrackerMapper` for the TET.
I refactored this in the last commit: https://github.com/dhis2/dhis2-core/pull/16880/commits/d921f3d4b014bde82d42d636525046b8fdfdd77c

Reply:
Perfect! Now it's clear that we map the sharing for metadata. Before I thought we map it now for data which we do not do elsewhere.